gui/wxpython/core/globalvar.py

```diff
@@ -263,7 +245,7 @@ def UpdateGRASSAddOnCommands(eList=None):
 
 """@Check version of wxPython, use agwStyle for 2.8.11+"""
 hasAgw = CheckWxVersion([2, 8, 11, 0])
-wxPython3 = CheckWxVersion([3, 0, 0, 0])
+wxPython3 = CheckWxVersion(WXPY3_MIN_VERSION)
```

Comment:
Since this is obsolete, it would be best to remove it completely. It's used in mapdisp/frame.py and mapswipe/frame.py, it should be easy to remove. It would be nice if you could incorporate it here, or I can then make another PR.

Reply:
Yes of course I added commit it in that PR.